index.html

```diff
@@ -72,13 +72,32 @@ <h2>Everyone needs a little Karma</h2>
           <div class="karma_icon_direction">
             <img
               class="karma_icon_images"
+              style="padding-left: 1rem"
               src="img/icon-refill.svg"
               alt="Image of a fuel pump"
             />
             <p>Pay as You Go</p>
           </div>
         </div>
       </section>
+      <section class="karma_feature">
+        <div>
+          <img
+            class="karma_feature_image"
+            src="additional/feature-img.png"
+            alt="Image of women sitting down and smiling whilst using a tablet"
+          />
+        </div>
+        <div class="wherever_i_am">
+          <p>
+            <em>
+              <i>"</i>Wherever I am, I just don't worry about my connection
```

Comment:
There's an element called `q` that can be used for the quotes.

Reply:
Yes, but I only want the quotation marks to turn orange and not the whole quote. I tried doing this with q but couldn't figure it out.

Reply:
Figured it out.